### PR TITLE
use non const message_t because of CI

### PR DESCRIPTION
**Pull request by xlqian**
our CI doesn't have "size() const" :(

### CODE REVIEW

File: zmq.cpp [hunk 1]
```diff
@@ -69,27 +63,34 @@ void LoadBalancer::run() {
...
+                if (nb_frames < 3) {
+                    frames[nb_frames].move(&frame);
+                }
                 // Are there more frames coming?
                 clients.getsockopt(ZMQ_RCVMORE, &more, &more_size);
+                nb_frames++;
             } while (more);
 
-            if (!is_valid_message(frames)) {
+            if (nb_frames > 3 || frames[1].size() != 0 ) {
```
Comment (AurelienLP) [inline zmq.cpp]:
```suggestion
            if (nb_frames > 3 || frames.at(1)].size() != 0 ) {
```

File: zmq.cpp [hunk 2]
```diff
@@ -16,13 +17,6 @@ std::string z_recv(zmq::socket_t& socket) {
     return std::string(static_cast<char*>(msg.data()), msg.size());
 }
 
-namespace {
-
-bool is_valid_message(const std::vector<zmq::message_t>& frames) {
```
Comment (AurelienLP) [inline zmq.cpp]: You removed this method because of the const?

Comment (xlqian) [reply zmq.cpp]: yep, it's a small funtion..

Comment (xlqian) [reply zmq.cpp]: and i hate non const...

**Pull request merged**